js/redirect-app.js

```diff
@@ -0,0 +1,13 @@
+// TODO: 
+// - Add dynamic URL's language?
+
+(function () {
+  if(navigator.userAgent.toLowerCase().indexOf("android") > -1){
+    window.location.href = 'https://play.google.com/store/apps/details?id=nl.rijksoverheid.en';
+  }
+  if(navigator.userAgent.toLowerCase().indexOf("iphone") > -1){
+    window.location.href = 'https://apps.apple.com/nl/app/id1517652429';
+  }
+
+  window.location.href = "/";
```

Comment:
Dit betekent dat `/app` _altijd_ doorgetikt wordt. Is dat de bedoeling voor deze hele pagina? (En zo ja, waarom heeft de pagina dan überhaupt content?)

Reply:
Volgens mij klopt dit. Hij heeft geen content meer nu toch?

Reply:
De pagina is een witte pagina zonder content. Een redirect na de app's of home. Eventueel later kan er een pagina van worden gemaakt. Met de badges of meta tags.